lib/reline/line_editor.rb

```diff
@@ -179,6 +179,14 @@ def handle_signal
     @rendered_screen.lines = []
     @rendered_screen.cursor_y = 0
     render_differential
+    case @old_winch_trap
+    when 'DEFAULT', 'SYSTEM_DEFAULT', 'IGNORE'
+      # Do nothing
+    when 'EXIT'
+      exit
+    else
+      @old_winch_trap.call if @old_winch_trap.respond_to?(:call)
+    end
```

Comment:
I think `@old_winch_trap.call if @old_winch_trap.respond_to?(:call)` is enough.

I don't think `Signal.trap('WINCH', 'EXIT')` will be used. Even if it is set, we can think that Reline override and ignore it.
We can consider adding `when 'EXIT'; exit` when there is a request with a use case.